docky/cmd/run_open.py

```diff
@@ -53,7 +52,7 @@ class DockyRun(DockyExec):
     """Start services and enter in your dev container"""
 
     def _check_running(self):
-        if self.project.get_containers(service=self.service):
+        if docker.compose.ps(services=[self.service], all=True):
```

Comment:
why all ?

Reply:
if you do not pass all, you do not see the odoo container as it launch with "run". All is to show all container even if there are not launch with the normal way  "docker compose up"

Reply:
but will you get exited containers ?

Reply:
Note: in the previous version we had the same behavior

And normally you should never have exited container for the main container (as we launch it with --rm).

**But** if you have used manually `docker compose run` and you didn't have launch `docker compose rm`
then you have an exited container and `docky run` is blocked !

So we can first remove this existed container (as it will stay here indefinitely) and then start the container correctly. See commit : https://github.com/akretion/docky/commit/0c60eebc1ed2321045a3ce234ad09fb532c022b7